### PR TITLE
Reduce default Maint. Drone playtime requirement

### PR DESCRIPTION
**Pull request by JJRcop**
## About The Pull Request

I have reduced the default Drone hours requirement from 40 to 14. This PR shouldn't be merged until DRONE_ROLE_PLAYTIME is set to 40 on the live game configs (currently unset), or a reasonable amount of time has passed and it is still unset.

I have no intention of slipping this behind anyone's back, and want to give ample notice to current admins.

## Why It's Good For The Game

I think it is a more reasonable default than 40, if even still on the higher side. 40 is a bit more of a nuclear setting based on fear from drones' previous incarnation.

## Changelog
:cl: JJRcop
server: Default Drone hours without config set reduced from 40 to 14. Please set DRONE_ROLE_PLAYTIME to 40 if you wish to retain old behavior.
/:cl:

### CODE REVIEW

**Comment by JJRcop** [comment]
I am open to discussion on modifying the number even further, or even the value of DRONE_REQUIRED_ROLE, but would like to remind everyone that this is about the codebase default only.

Discussion about the values on the /tg/station13 servers should not be discussed here but rather with current headmins.

**Comment by Timberpoes** [comment]
Headmins have been requested to make sure the configs are set to appropriate values that they are happy with before this PR is merged, as requested by the PR author.

**Comment by Timberpoes** [comment]
DNM until the headmins have made the necessary config additions or have stated they are happy to use the 14h default instead of 40h.

**Comment by MrStonedOne** [comment]
configs that are not defined/overridden in the server should be treated as configs the headmins/host are uninterested in currently overriding.

ie: changing default config values is the kind of thing that i would expect coderbus be allowed  to do without waiting for a headmin response.

This is the kind of thing that deserves a ping along the lines of:

`we are changing the config default of x to 14 hours after feedback that it was too high. if you haven't defined this config yet it will use the new default, otherwise it will use your defined value`

**Comment by Timberpoes** [comment]
> 
> 
> configs that are not defined/overridden in the server should be treated as configs the headmins/host are uninterested in currently overriding.
> 
> ie: changing default config values is the kind of thing that i would expect coderbus be allowed to do without waiting for a headmin response.
> 
> This is more the kind of thing that deserves a ping along the lines of:
> 
> `we are changing the config default of x to 14 hours after feedback that it was too high. if you haven't defined this config yet it will use the new default, otherwise it will use your defined value, please make any changes before the pr hits 24 hours and becomes eligible for merging`

@MrStonedOne I am mindful of the fact the re-introduction of ghost role drones was done 28 days ago (https://github.com/tgstation/tgstation/pull/58249) and the PR was blocked pending the headmin team outlining their requirements to reintroduce this feature. One of their requirements in being okay with the re-introduction of drones was a 40 hour silicon playtime requirement.

The following set of guidelines are pinned in policybus and include all the factors that had to be agreed from a policy side to get drones re-introduced.

image

I would prefer to get a headmin response that they've set the appropriate config values before merging this change. As far as I am aware, their position of requiring 40 silicon hours to play has not changed.

If you feel this is something that is capable of being merged without going against the spirit of what the headmins agreed with JRRcop then I'll un-DNM tag it.

**Review by Timberpoes — APPROVED** [review]

**Comment by MrStonedOne** [comment]
image

**Comment by optimumtact** [comment]
did not read

**Comment by dondonpow** [comment]
holy shit thank you.

**Pull request merged**
